v4/internal/otel_expect.go

```diff
@@ -0,0 +1,40 @@
+package internal
+
+import (
+	"go.opentelemetry.io/otel/api/trace/testtrace"
+)
+
+// OpenTelemetryExpect implements internal.Expect for use in testing.
+type OpenTelemetryExpect struct {
+	*testtrace.Tracer
+}
+
+// ExpectTxnEvents TODO
+func (e *OpenTelemetryExpect) ExpectTxnEvents(t Validator, want []WantEvent) {}
+
+// ExpectSpanEvents TODO
+func (e *OpenTelemetryExpect) ExpectSpanEvents(t Validator, want []WantEvent) {}
```

Comment:
Currently all the expect methods do nothing. I chose to do this work in stages. The next Pull Request I plan on opening will implement `ExpectTxnEvents` and `ExpectSpanEvents`. I can see we may consider implementing `ExpectTxnMetrics` as well, but since we aren't actually going to be creating any metrics, I'm a bit iffy on it.